test/AutoDiff/validation-test/forward_mode.swift

```diff
@@ -1319,4 +1319,21 @@ ForwardModeTests.test("ForceUnwrapping") {
   expectEqual(5, forceUnwrap(Float(2)))
 }
 
+//===----------------------------------------------------------------------===//
+// Array methods from ArrayDifferentiation.swift
+//===----------------------------------------------------------------------===//
+ForwardModeTests.test("Array.differentiableMap") {
```

Comment:
minor: please add a empty newline after the comment
```suggestion
//===----------------------------------------------------------------------===//

ForwardModeTests.test("Array.differentiableMap") {
```